models/src_apple_store.yml

```diff
@@ -2,7 +2,7 @@ version: 2
 
 sources:
   - name: apple_store # This source will only be used if you are using a single apple_store source connector. If multiple sources are being unioned, their tables will be directly referenced via adapter.get_relation.
-    schema: "{{ var('apple_store_schema', 'itunes_connect') }}"
+    schema: "{{ var('apple_store_schema', 'apple_appstore') }}"
```

Comment:
Is this accurate? This is actually a very important piece for Quickstart to work. The default schema **must** be what the default schema provided by the connector. In this case it looks like it's still `itunes_connect`.
![image](https://github.com/user-attachments/assets/f8f0f5bf-1c30-40f6-9935-77867aba9d07)

We need to keep that default schema.